odoo/sql_db.py

```diff
@@ -136,10 +136,10 @@ class BaseCursor:
     """ Base class for cursors that manage pre/post commit hooks. """
 
     def __init__(self):
-        self.precommit = tools.Callbacks()
```

Comment:
Can you `from odoo.tools.misc import Callabacks`?
It avoids 2 lookups on every cursor creation.

Reply:
I second @kmagusiak 's suggestion.